app/models/FileUploads.scala

```diff
@@ -27,6 +27,10 @@ case class FileUploads(files: Seq[FileUpload] = Seq.empty) {
   def nonEmpty: Boolean = !isEmpty
   def isSingle: Boolean = acceptedCount == 1
 
+  def initiateCount: Int =
```

Comment:
feels to me like this should be a method that returns a boolean, with a different name for the method.

You dont really care what the count is, you want to know if the file has been initiated.